Makefile

```
@@ -360,6 +374,26 @@ $(MOCKSERVER_BIN): $(LOCALBIN)
 metalctl: $(METALCTL) ## Build metalctl locally if necessary.
 	go build -o $(METALCTL) ./cmd/metalctl
 
+.PHONY: promtool
+promtool: $(PROMTOOL) ## Download promtool locally if necessary.
+$(PROMTOOL): $(LOCALBIN)
```

checkmake (0.3.2)

[warning] Target body for "$(PROMTOOL)" exceeds allowed length of 5 lines (11).

(maxbodylength)